### PR TITLE
chore: add development environment

### PR DESCRIPTION
**Pull request by jarqvi**
### Author should do the followings, if applicable

- [x] Add tests
- [x] Run tests
- [x] `yarn denoify` to generate files for Deno

### CODE REVIEW

**Comment by jarqvi** [comment]
I recently familiarized myself with your project. Having thoroughly reviewed the project documentation, I am enthusiastic about contributing to its development.

To kick things off, given my keen interest in Docker container development, I've implemented some modifications to facilitate a smoother onboarding process for newcomers to the project.

**Comment by yusukebe** [comment]
Welcome! @jarqvi 

I want to merge this PR. Check my comment, fix it, and please do `yarn denoify`.

**Comment by jarqvi** [comment]
Thanks for Notes.
Done.

File: README.md [hunk 1]
```diff
@@ -100,6 +100,14 @@ Yusuke Wada <https://github.com/yusukebe>
 
 _RegExpRouter_, _SmartRouter_, _LinearRouter_, and _PatternRouter_ are created by Taku Amano <https://github.com/usualoma>
 
+## Local Development
```
Comment (yusukebe) [inline README.md:103]: Could you write these lines to [`docs/CONTRIBUTING.md`](https://github.com/honojs/hono/blob/main/docs/CONTRIBUTING.md)? We don't have to edit README for these lines.

**Comment by yusukebe** [comment]
@jarqvi 

Sorry! I've noticed one thing that I've commented on.

**Comment by jarqvi** [comment]
it's good idea, ok.

**Review by yusukebe — APPROVED** [review]
LGTM!

**Pull request merged**